### PR TITLE
#167164941 Reset Password Component

### PR DESCRIPTION
**Pull request by Ukhu**
#### Pivotal tracker story
[#167164941](https://www.pivotaltracker.com/story/show/167164941)

#### What does this PR do?
   - Create component for reset password

#### Summary of Task
   - Design reusable components and style with SCSS
   - Write logic for interacting with component

#### How can this be tested?
   - pull the branch to your local repository: `git pull origin/feature/167164941/reset-password-component`
   - install packages and dependencies: `npm install`
   - run `npm run dev`
   - visit the root path of your local server

### CODE REVIEW

**Review by WilliamsOhworuka — APPROVED** [review]

File: src/components/App.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+import ResetPassword from './ResetPassword/ResetPassword';
+import './app.scss';
 
 const Welcome = () => (
-  <div className={style.app}>
+  <div className="app">
+    <ResetPassword />
     <h1>Authors Haven</h1>
-    <div className={style.sassyDiv}>Welcome!</div>
+    <div className="sassyDiv">Welcome!</div>
```
Comment (Nkemjiks) [inline src/components/App.js]: Take this out

File: src/components/AuthModal/AuthModal.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './AuthModal.scss';
+
+const AuthModal = ({
+  children, title, desc, isOpen,
+}) => {
+  const [display, setDisplay] = useState(isOpen);
```
Comment (Nkemjiks) [inline src/components/AuthModal/AuthModal.js]: move to a separate js file

File: src/components/AuthModal/AuthModal.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
+.backdrop-div {
```
Comment (Nkemjiks) [inline src/components/AuthModal/AuthModal.scss]: Wrap you up a parent class

File: src/components/ResetPassword/ResetPassword.js [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+import axios from 'axios';
+import validator from 'validator';
+import AuthModal from '../AuthModal/AuthModal';
+import TextInput from '../TextInput/TextInput';
+import SuccessIcon from '../Icons/SuccessIcon';
+import appConfig from '../../config/appConfig';
+import './ResetPassword.scss';
+
+const ResetPassword = () => {
+  const title = 'FORGOT PASSWORD';
```
Comment (Nkemjiks) [inline src/components/ResetPassword/ResetPassword.js]: Move from here to line 99 to a separate js file

File: src/components/ResetPassword/ResetPassword.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.reset-password-form {
+    
+    .validation-error {
```
Comment (Nkemjiks) [inline src/components/ResetPassword/ResetPassword.scss]: Format the css properly

**Review by Nkemjiks — CHANGES_REQUESTED** [review]

File: src/components/AuthModal/AuthModal.scss (hunk 3 above)
Comment (Ukhu) [reply src/components/AuthModal/AuthModal.scss]: The `backdrop-div` class is on the parent `div` of the `AuthModal` component. Perhaps I should rename the class to `auth-modal-div`?

**Comment by Nkemjiks** [comment]
Looks good but i need some minor changes 

- Can you make this more descriptive `you will receive a link in your mail shortly`. You can check for samples online

- The banner gradient is only covering half of the screen. Might not be coming from you but can you fix it
Screenshot 2019-09-05 at 7 33 08 AM

**Review by OvieMudi — CHANGES_REQUESTED** [review]
Password reset page is undefined. Please review.

**Review by Nkemjiks — APPROVED** [review]

**Pull request merged**